src/main/java/org/kohsuke/github/GitUser.java

```diff
@@ -15,10 +15,10 @@
 @SuppressFBWarnings(value = { "UWF_UNWRITTEN_PUBLIC_OR_PROTECTED_FIELD", "UWF_UNWRITTEN_FIELD", "NP_UNWRITTEN_FIELD" },
         justification = "JSON API")
 public class GitUser {
-    private String name, email, date;
+    private String name, email, date, username;
 
     /**
-     * Gets name.
+     * Gets name (e.g. from git commit).
```

Comment:
The commit aspect is mentioned on line 8 above.  I see you're trying to be clear.  Maybe the suggestion below?

```suggestion
     * Gets the git user name for an author or committer on a git commit.
```